core/src/mindustry/world/blocks/ConstructBlock.java

```diff
@@ -288,6 +289,9 @@ public void deconstruct(Unit builder, @Nullable Building core, float amount){
             }
 
             progress = Mathf.clamp(progress - amount);
+            
+            // fast forward boulder deconstruction when visually already gone
+            if(progress <= 0.35f && previous instanceof Boulder) progress = 0;
```

Comment:
Instead of hardcoding it for boulders, the number should be a special property that is 0 for normal blocks.

Reply:
ok, I'll change it within the hour 👍

Reply:
any name suggestions?

- deconstructionFinishesAt
- deconstructTrigger
- demolishThreshold

they just don't sound quite right 🤔

Reply:
`deconstructThreshold` seems appropriate